mmdet/utils/benchmark.py

```diff
@@ -160,7 +160,6 @@ def __init__(self,
         print_log('before build: ', self.logger)
         print_process_memory(self._process, self.logger)
 
-        self.cfg.model.pretrained = None
```

Comment:
Instead of deleting it directly, we should first determine if the attribute is present.